include/swift/Runtime/Config.h

```diff
@@ -17,8 +17,34 @@
 #ifndef SWIFT_RUNTIME_CONFIG_H
 #define SWIFT_RUNTIME_CONFIG_H
 
-// Bring in visibility attribute macros for library visibility.
-#include "llvm/Support/Compiler.h"
+/// \macro SWIFT_RUNTIME_GNUC_PREREQ
+/// Extend the default __GNUC_PREREQ even if glibc's features.h isn't
+/// available.
+#ifndef SWIFT_RUNTIME_GNUC_PREREQ
+# if defined(__GNUC__) && defined(__GNUC_MINOR__) && defined(__GNUC_PATCHLEVEL__)
```

Comment:
Are we up to the point where the runtime is always built with the just-built Clang? If so, we can drop all this too.

Reply:
Yes, we always use the just-built Clang to build the runtime. We've discussed changes to allow building the stdlib and runtime separately (where there may not be a just-built Clang), but in that case, if someone chooses to use that option (in some future version where it is supported), I think we will then require the system compiler to be a suitably recent version of Clang.